frontend.conf

```diff
@@ -33,4 +46,106 @@ server {
     }
 }
 
+# -----------------------------------------------------------------------------#
+#                                                                              #
+# 2. Advanced Example: Landing page, login/logout button to handle OIDC kflow  #
+#                                                                              #
+#  - Landing page shows 'login' button                                         #
+#  - 'login' button calls `/login` endpoint to start OIDC flow by validating 
+#    'id_token' w/ IdP's JWK.  #
+#  - Landing page calls `/userinfo` to show user info using 'access_token`.    #
+#  - 'logout' button to be finished OIDC session by IdP.                       #
+#  - API authorization by validating `access_token` w/ IdP's JWK               #
+#                                                                              #
+# -----------------------------------------------------------------------------#
+
+#
+# Upstream server for proxing to the frontend site.
```

Comment:
- Yes, you are right! 
- The purpose is for customers to easily follow the example of demo which we are going to create the feature branch (e.g, `demo`) instead of the 3rd-party link as discussed. 
- Given the kind of demo, customers can quickly set up IdP, configure NGINX Plus, and develop their application using NGINX Plus OIDC.
- Therefore, I just added the link as the `demo` branch is not created yet.
- However, I will remove the link as for now. 
- Let me know once you create a feature branch such as `demo` so that I can add it to there.